app/build.gradle

```diff
@@ -23,9 +23,6 @@ apply plugin: 'androidx.navigation.safeargs.kotlin'
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
-    buildFeatures {
-        dataBinding true
-    }
```

Comment:
Revert. Some layouts still use data binding.

Reply:
Is it need to have two buildFeatures in Gradle and two dataBinding true?

Reply:
Oh, gotcha. I missed your comment that this is a duplicate.